maliput_ros/include/maliput_ros/ros/maliput_query.h

```diff
@@ -53,6 +55,20 @@ class MaliputQuery final {
   /// @return The maliput::api::RoadGeometry.
   inline const maliput::api::RoadGeometry* road_geometry() const { return road_network_->road_geometry(); }
 
+  /// Finds a maliput::api::Junction by its ID.
+  /// @param[in] id The maliput::api::JunctionId.
+  /// @return A maliput::api::Junction when @p id refers to a valid maliput::api::Junction. Otherwise, nullptr..
+  inline const maliput::api::Junction* GetJunctionBy(const maliput::api::JunctionId& id) const {
+    return road_network_->road_geometry()->ById().GetJunction(id);
+  }
+
+  /// Finds a maliput::api::Segment by its ID.
+  /// @param[in] id The maliput::api::SegmentId.
+  /// @return A maliput::api::Segment when @p id refers to a valid maliput::api::Segment. Otherwise, nullptr..
```

Comment:
nit: Typo. Doble period at the end of the sentence.

Reply:
Done.